README.md

```diff
@@ -1,20 +1,48 @@
 
-## Installation
+## Features
+- **Ease of use:** Code Verify is applied directly in browser. It consistently checks for insecurities and modifications, and will only alert you if a problem arises with a popup.
+- **Download source code directly:** If there is a problem with a web page you visit, you can download the source code directly through Code Verify to see what kind of modifications could have been made.
+- **Automatic updates:** Code verify updates as the source code for its supported web pages do. For example, if WhatsApp recieves a new feature and its code is modified, users will not be alerted and Code Verify automatically accomodates for this.
 
-You can install Code Verify from the extension store of [Chrome](https://chrome.google.com/webstore/detail/code-verify/llohflklppcaghdpehpbklhlfebooeog), [Firefox](https://addons.mozilla.org/en-US/firefox/addon/code-verify/#:~:text=The%20new%20Code%20Verify%20is,inauthentic%20or%20has%20been%20modified.), or [Edge](https://microsoftedge.microsoft.com/addons/detail/code-verify/cpndjjealjjagamdecpipjfamiigaknk#:~:text=Code%20Verify%20will%20immediately%20alert,and%20hasn't%20been%20modified.). (Safari support coming soon)
+## Installation Steps
+- You can install Code Verify from the extension store of [Chrome](https://chrome.google.com/webstore/detail/code-verify/llohflklppcaghdpehpbklhlfebooeog), [Firefox](https://addons.mozilla.org/en-US/firefox/addon/code-verify/#:~:text=The%20new%20Code%20Verify%20is,inauthentic%20or%20has%20been%20modified.), or [Edge](https://microsoftedge.microsoft.com/addons/detail/code-verify/cpndjjealjjagamdecpipjfamiigaknk#:~:text=Code%20Verify%20will%20immediately%20alert,and%20hasn't%20been%20modified.). (Safari support coming soon)
+- Once installed, navigate to your extensions manager and update your permissions according to what websites you wish to use Code Verify on.
+- With Code Verify up and running, you can ensure that your browsing on WhatsApp, Instagram, Facebook, and Messenger are secure.
 
-### [Code of Conduct](https://code.fb.com/codeofconduct)
+## Contributing
+We are open to community solutions and updates. If you would like to contribute features, source code, or have any other issues to report, consult [CONTRIBUTING.md](CONTRIBUTING.md) and follow the guidelines posted. 
 
-Meta has adopted a Code of Conduct that we expect project participants to adhere to. Please read [the full text](https://code.fb.com/codeofconduct) so that you can understand what actions will and will not be tolerated.
+## Contributors
+We very much appreciate the contributions made to Code Verify by the following individuals:
 
-### License
+- [ezzak](https://github.com/ezzak)
+- [rich-hansen](https://github.com/rich-hansen)
+- [aselbie](https://github.com/aselbie)
+- [KarimP](https://github.com/KarimP)
+- [dneiter](https://github.com/dneiter)
+- [m-lyons](https://github.com/m-lyons)
```

Comment:
Can we leave this section out please?

Or link to `https://github.com/facebookincubator/meta-code-verify/graphs/contributors` if necessary